packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/logical-assignment/options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": [
+    "proposal-logical-assignment-operators",
```

Comment:
Can you add `loose: true` to both plugins?
```json
{
  "plugins": [
    ["proposal-logical-assignment-operators", { "loose": true }],
    ["proposal-class-properties", { "loose": true }]
  ]
}
```
The `private-loose` folder is test cases about `loose` options so we are sure that both `loose` and non-loose (default) are working.